docker-compose-auditor.yml

```diff
@@ -73,6 +73,7 @@ services:
 
   scalar-auditor:
     image: ghcr.io/scalar-labs/scalar-auditor:4.0.0-SNAPSHOT
+    # image: 709825985650.dkr.ecr.us-east-1.amazonaws.com/scalar/scalar-auditor:4.0.0-SNAPSHOT
```

Comment:
Sorry, just my curiosity. Basically, is this part `709825985650 ` never change? Is there any stable name just like ghcr.io ...?

Reply:
@jnmt  
Thank you for your review!

> Sorry, just my curiosity. Basically, is this part `709825985650` never change? Is there any stable name just like ghcr.io ...?

Yes. The part `709825985650` is fixed value. It depends on the specifications of AWS Marketplace.
If you use Scalar products via AWS Marketplace, you can always use the same value `709825985650`.